### ISSUE TITLE
Any chance we can make xengine not have to restart after F engine reprogramming?

### ISSUE THREAD

**Issue by yupinghuang** [ovro-lwa/lwa-issues#440]
x engine pipelines typically shows rx gpbs of 0 after f engine programming/initialization.

Feel free to close if it's a hassle. It's a nice-to-have.

**Comment by jack-h** [comment]
You're right that this is worth having. I'll need some engineering time to
try and figure out what the deal is on the X-engine side. In theory
F-engine restarts should be tolerated, but I guess fails in real life
